Rancher-mac.md

```diff
@@ -0,0 +1,32 @@
+# Switching to Rancher Desktop on MacOS
+
+## Removing old entries
+
+- Uninstall Docker Desktop
+- Remove Existing Symlinks:
+
+```bash
+cd /usr/local/bin
+rm docker docker-compose-v1 docker-credential-desktop docker-credential-ecr-login docker-credential-osxkeychain com.docker.cli
+cd ~
+```
+
+- In ~/.docker/config.json rename **credsStore** to **credStore**
+
+## Installing Rancher Desktop
+
+- Download [Rancher Desktop](https://github.com/rancher-sandbox/rancher-desktop/releases)
```

Comment:
Specify:
- not to download a beta but a stable version
- which one of the downloads (as the mac variant has several different variations)